code/datums/martial/holy_crunch.dm

```diff
@@ -0,0 +1,40 @@
+/datum/martial_art/holy_crunch
+	name = "Holy Crunch"
+	deflection_chance = 5
+	no_guns = TRUE
+	allow_temp_override = FALSE
+
+/datum/martial_art/chaplain_snap/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("power word: grail", "power word: justice", "power word: purge", "power word: judicator", "power word: banish")
+	D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+	D.apply_damage(10, BRUTE)
```

Comment:
ignores armor??

Reply:
there are 18 damage chaplain weapons that pierce armor so it doesn't sound that crazy

Reply:
Weapons that can be disarmed or have a tell via sprite or examine.

Reply:
As long as hulks and the officer's saber are a thing, job-specific unarmed attacks that do a lot of damage and pierce armor will always not be as good as them